src/locale/global.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * This file is automatically generated.
+ * Run 'pnpm run generate:locales' to update.
+ */
+
+import { Faker } from '../faker';
+import en from '../locales/en';
+import global from '../locales/global';
+
+export const faker = new Faker({
+  locale: 'global',
+  localeFallback: 'en',
+  locales: {
+    global,
+    en,
+  },
```

Comment:
This needs to be
```suggestion
  locale: [global],
```
after #1735 got merged

Reply:
or just `global`